.github/workflows/update-networks-submodule.yml

```
@@ -0,0 +1,95 @@
+          if [ -d networks/.git ]; then
+            git -C networks config user.name "github-actions[bot]"
+            git -C networks config user.email "github-actions[bot]@users.noreply.github.com"
+          fi
+
+      - name: Install Rust (reads rust-toolchain.toml)
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          components: rustfmt,clippy
+
+      - name: run cargo clean
+        uses: actions-rs/cargo@v1
```

actionlint (1.7.8)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)